src/kinetics/include/antioch/reaction.h

```diff
@@ -324,9 +324,9 @@ namespace Antioch
     std::vector<unsigned int> _reactant_stoichiometry;
     std::vector<unsigned int> _product_stoichiometry;
     std::vector<CoeffType> _efficiencies;
-    std::vector<unsigned int> _species_reactant_stoichiometry;
-    std::vector<unsigned int> _species_product_stoichiometry;
-    std::vector<int>          _species_delta_stoichiometry;
+//    std::vector<unsigned int> _species_reactant_stoichiometry;
```

Comment:
Personally I didn't want to futz too much with Sylvain's commits.  I'd vote to delete too.